sig-security/sig-security-external-audit/README.md

```diff
@@ -0,0 +1,65 @@
+# SIG Security External Audit Subproject
+
+## Overview
+
+The SIG Security External Audit subproject (subproject, henceforth) is responsible for coordinating regular, comprehensive, third-party security audits.
+The subproject publishes the deliverables of the audit after abiding to the [Security Release Process](https://github.com/kubernetes/security/blob/master/security-release-process.md) and [embargo policy](https://github.com/kubernetes/security/blob/master/private-distributors-list.md#embargo-policy).
+
+  - [Request for Proposal (RFP)](#rfp)
+    - [Security Audit Scope](#security-audit-scope)
+    - [Vendor and Community Questions](#vendor-and-community-questions)
+  - [Review of Proposals](#review-of-proposals)
+  - [Vendor Selection](#vendor-selection)
+  - [Deliverables](#deliverables)
+
+## RFP
+
+The subproject produces a RFP for a third-party, comprehensive security audit. The subproject publishes the RFP in the `kubernetes/community/sig-security` repository. The subproject defines the scope, schedule, methodology, selection criteria, and deliverables in the RFP.
```

Comment:
I don't think sig-security is its own repo. just a folder

```suggestion
The subproject produces a RFP for a third-party, comprehensive security audit. The subproject publishes the RFP in the 'sig-security' folder in the `kubernetes/community/` repository. The subproject defines the scope, schedule, methodology, selection criteria, and deliverables in the RFP.
```

Reply:
applied suggestion in commit 947b473